### DISCUSSION TITLE
Tune/Flow Rate - does not work

### DISCUSSION THREAD

**Discussion by transcendReality** [Jyers/Marlin#1142] [Q&A] [answered]
I see no changes in filament flow while printing on the fly and using the 'Tune' menu. I've increased by as much as 26%, and I see no difference. Using an Ender 3 V2

**Comment by Cal-LF** [comment]
Next print I will try it. I rarely use the flow adjustment.

**Comment by Cal-LF** [comment, marked as answer]
@transcendReality I printed this spacer to test the flow adjustment in the tune menu. The flow was raised to 150% and I can see where the increase in flow was started at layer 15 of 22. This indicates the flow adjustment does indeed work.

![_XZ25250](https://user-images.githubusercontent.com/76499230/125498541-e79dc1f1-dcf8-4b2a-ac22-c824474c7f7d.JPG)
